components/script/dom/webidls/ServiceWorkerContainer.webidl

```diff
@@ -6,21 +6,21 @@
 [Pref="dom.serviceworker.enabled", Exposed=(Window,Worker)]
 interface ServiceWorkerContainer : EventTarget {
   [Unforgeable] readonly attribute ServiceWorker? controller;
-  //[SameObject] readonly attribute Promise<ServiceWorkerRegistration> ready;
+  [SameObject] readonly attribute Promise<ServiceWorkerRegistration> ready;
 
   [NewObject, Throws] ServiceWorkerRegistration register(USVString scriptURL, optional RegistrationOptions options);
 
-  //[NewObject] /*Promise<any>*/ any getRegistration(optional USVString clientURL = "");
-  //[NewObject] /* Promise */<sequence<ServiceWorkerRegistration>> getRegistrations();
+  [NewObject] Promise<any> any getRegistration(optional USVString clientURL = "");
+  [NewObject] Promise <sequence<ServiceWorkerRegistration>> getRegistrations();
```

Comment:
We shouldn't be modifying any other methods or properties in this file for the purposes of these changes.